packages/tools-iottwinmaker/README.md

```diff
@@ -48,7 +53,14 @@ tmdt -h
 ___
 
 
-## Bootstrap a TMDT project from an existing workspace
+## TMDT Init
+
+### Init Parameters (required)
+- *--region*: Specify the AWS region of the Workspace to bootstrap the project from
+- *--workspace-id*: Specify the ID of the Workspace to bootstrap the project from
+- *--out*: Specify the directory to initialize a project in
+
+Use this command to bootstrap a TMDT project from an existing workspace.
 
 The following will initialize a tmdt project at the specified directory with a `tmdt.json` file
 
```

Comment:
"CookieFactory0601" and "SyncB" should be replaced with something more generic, maybe just "YourWorkspace" ?

Same for "testproj"

Reply:
makes sense